Rasa_Bot/actions/actions_minimum_functional_product.py

```diff
@@ -175,3 +178,38 @@ async def run(self, dispatcher, tracker, domain):
                                     "This is a tracker",
                                     recursive_rule)
         return[]
+
+
+class ValidateActivityGetFileForm(FormValidationAction):
+    def name(self) -> Text:
+        return 'validate_activity_get_file_form'
+
+    def validate_received_file_text(
+            self, slot_value: Text, dispatcher: CollectingDispatcher,
+            tracker: Tracker, domain: Dict[Text, Any]) -> Dict[Text, Any]:
+        # pylint: disable=unused-argument
+        """Validate the presence of an attached file"""
+
+        # here the message metadata are retrieved
+        events = tracker.current_state()['events']
+        user_events = []
+        for e in events:
```

Comment:
```suggestion
        user_events = [e for e in events if e['event'] == 'user']
```
Would something like this be more readable? (actually maybe not...)